docs/plugins.md

```diff
@@ -0,0 +1,262 @@
+[runs an audit](https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules)
+locally by injecting the Dev Tools script into WebDriver pages, and it can diagnose issues including
+missing labels, incorrect ARIA attributes and color contrast. This is a great starting point if
+you can't send source code over the wire through an API.
+
+[Tenon.io](http://www.tenon.io) has a more robust set of tests to help you find
+accessibility issues, but it requires [registering](http://tenon.io/register.php) for an API key
+and making an external request for each test, which may not work for everyone. Some people use
+Tenon with introspection services like ngrok or localtunnel to securely
+test local web servers. Protractor takes the [options you provide](http://tenon.io/documentation/understanding-request-parameters.php) in the plugin configuration and sends them
+with the page source to the Tenon API. One limitation of this approach is that all scripts must be reachable from the page source as a string, for example, by using a CDN.
+For projects with an MIT license, Tenon is free but with a limited
+daily API limit. Paid subscriptions are available for enterprise and commercial projects.
+
+Enable this plugin in your config file:
+```js
+  // Chrome Accessibility Dev Tools only:
+  exports.config = {
+      ...
+      plugins: [{
+        chromeA11YDevTools: {
+          treatWarningsAsFailures: true
+        },
+        path: 'node_modules/protractor.plugins/accessiblity'
+      }]
+    }
+```
+```js
+  // Tenon.io only:
+  exports.config = {
+      ...
+      plugins: [{
+        tenonIO: {
+          options: {
+            // See http://tenon.io/documentation/understanding-request-parameters.php
+            // options.src will be added by the test.
+          },
+          printAll: false, // whether the plugin should log API response
+        },
+        chromeA11YDevTools: true,
+        path: 'node_modules/protractor/plugins/accessiblity'
+      }]
+    }
+```
+
+ngHint Plugin
+-------------
+ngHint adds run-time hinting to AngularJS projects. This plugin bubbles those hints up to the
+command line so they can be used in automated testing.
+
+You enable this plugin in your config file:
+ ```js
+exports.config = {
+  plugins: [{
+    path: 'node_modules/protractor/plugins/ngHint',
+
+    asTests: {Boolean},
+    excludeURLs: {(String|RegExp)[]}
+  }]
+};
+```
+`asTests` specifies if the plugin should generate passed/failed test results
+based on the ngHint output or instead log the results to the console.
+Defaults to true.
+
+`excludeURLs` specifies a list of URLs for which ngHint results should be
+ignored. Defaults to []
+
+The strategy for this plugin is as follows:
```

Comment:
This like though line 238 are about the internal implementation.  Not sure it's relevant to this doc